tests/modules/services/shikane/basic-configuration.nix

```
@@ -0,0 +1,47 @@
+{ config, pkgs, ... }: {
```

The `pkgs` argument doesn't seem to be referenced:

```suggestion
{ config, ... }: {
```